packaging/ubuntu-14.04/rules

```diff
@@ -120,7 +120,8 @@ override_dh_auto_build:
 	mkdir -p _build/src/$(DH_GOPKG)/cmd/snap/test-data
 	cp -a cmd/snap/test-data/*.gpg _build/src/$(DH_GOPKG)/cmd/snap/test-data/
 	cp -a bootloader/assets/data _build/src/$(DH_GOPKG)/bootloader/assets
-	dh_auto_build -- $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS)
+	GOINVOKEFLAGS='-mod=vendor'	GO111MODULE=on \
+		dh_auto_build -- -mod vendor $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS) $(DH_GOPKG)/cmd/...
```

Comment:
previously this was not specifying the `./cmd/...` directory, is this intentional and also is that important for 14.04 specifically?

Reply:
Yes, we build with go modules, so go is expected to be invoked from the main module directory, but since GOPATH is set, we can use the import path to invoke it and do the right thing.